lrpar/src/lib/ctbuilder.rs

```diff
@@ -1249,11 +1264,6 @@ where
     }
 }
 
-/// Return a version of the string `s` which is safe to embed in source code as a string.
-fn str_escape(s: &str) -> String {
```

Comment:
This function was added here:
https://github.com/softdevteam/grmtools/commit/87a16197f54d9bdba130991d4ba2bba5333105f9

But I believe that the tests that were added in that patch now live here:
https://github.com/softdevteam/grmtools/blob/master/lrpar/cttests/src/epp.test

So I believe that `quote!` does the right thing here.